tka/tailchonk.go

```diff
@@ -176,8 +176,11 @@ updateLoop:
 //
 // FS implements the Chonk interface.
 type FS struct {
-	base string
-	mu   sync.RWMutex
+	mu sync.RWMutex
+
+	// Base is the root directory where AUMs will be stored.
+	// The dir will be created automatically if it does not already exist.
+	Base string
```

Comment:
Having `Base` as an exported field is a bit weird, since now callers can change it in the middle of using `FS`. The comment says that the directory will be created, but it's unclear if `Base` can be an empty string.

I don't feel too strongly about this, but my preference would be to keep `base` private and add a `NewFS` function that would create an `FS`, checking that it's not empty.

Reply:
Good idea, and now I look there's already a `ChonkDir()` function I can use.